mathics/builtin/drawing/image_internals.py

```diff
@@ -63,7 +63,7 @@ def pixels_as_float(pixels):
         return pixels.astype(numpy.float32) / 255.0
     elif dtype == numpy.uint16:
         return pixels.astype(numpy.float32) / 65535.0
-    elif dtype == numpy.bool:
+    elif dtype == bool:
```

Comment:
BTW this entire module should be moved to `mathics.eval`.

Reply:
BTW: shouldn't be ```dtype is bool``` ?

Reply:
Yes that is correct. I have made the change in b0a324fb